Fri Sep  9 01:49:55 2022 - INFO  - Start environmental preparation.
Failed to stop firewalld.service: Unit firewalld.service not loaded.
Failed to disable unit: Unit file firewalld.service does not exist.
passwd: Unknown user name 'hacluster'.
Fri Sep  9 01:50:39 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Fri Sep  9 01:50:40 2022 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r /etc/hosts @:/etc/
ssh: Could not resolve hostname : Name or service not known
lost connection
Fri Sep  9 01:50:41 2022 - ERROR - Failed in remote SCP operation: 1
Fri Sep  9 01:50:42 2022 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r /etc/corosync/corosync.conf @:/etc/corosync/
ssh: Could not resolve hostname : Name or service not known
lost connection
Fri Sep  9 01:50:43 2022 - ERROR - Failed in remote SCP operation: 1
Failed to start pcsd.service: Unit pcsd.service not found.
Failed to start pacemaker.service: Unit pacemaker.service not found.
../common/ha.sh: line 83: pcs: command not found
../common/ha.sh: line 84: pcs: command not found
../common/ha.sh: line 85: crm_verify: command not found
Job for corosync.service failed because the control process exited with error code.
See "systemctl status corosync.service" and "journalctl -xeu corosync.service" for details.
Fri Sep  9 01:50:46 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
../common/ha.sh: line 98: pcs: command not found
Failed to restart pacemaker.service: Unit pacemaker.service not found.
Job for corosync.service failed because the control process exited with error code.
See "systemctl status corosync.service" and "journalctl -xeu corosync.service" for details.
Failed to restart pcsd.service: Unit pcsd.service not found.
Fri Sep  9 01:50:49 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Fri Sep  9 01:50:58 2022 - INFO  - pkgs:(corosync-qdevice corosync-qnetd) is already installed
Fri Sep  9 01:51:00 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Fri Sep  9 01:51:03 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Fri Sep  9 01:51:04 2022 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r /etc/hosts @:/etc/
ssh: Could not resolve hostname : Name or service not known
lost connection
Fri Sep  9 01:51:05 2022 - ERROR - Failed in remote SCP operation: 1
Fri Sep  9 01:51:06 2022 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r /etc/hosts @:/etc/
ssh: Could not resolve hostname : Name or service not known
lost connection
Fri Sep  9 01:51:07 2022 - ERROR - Failed in remote SCP operation: 1
Fri Sep  9 01:51:10 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
oe_test_service_corosync-qnetd.sh: line 39: pcs: command not found
oe_test_service_corosync-qnetd.sh: line 40: pcs: command not found
Creating /etc/corosync/qnetd/nssdb
Creating new key and cert db
password file contains no data
Creating new noise file /etc/corosync/qnetd/nssdb/noise.txt
Creating new CA


Generating key.  This may take a few moments...

Is this a CA certificate [y/N]?
Enter the path length constraint, enter to skip [<0 for unlimited path]: > Is this a critical extension [y/N]?


Generating key.  This may take a few moments...

Notice: Trust flag u is set automatically if the private key is present.
QNetd CA certificate is exported as /etc/corosync/qnetd/nssdb/qnetd-cacert.crt
Fri Sep  9 01:51:18 2022 - INFO  - End of environmental preparation!
Fri Sep  9 01:51:19 2022 - INFO  - Start testing...
     Active: active (running) since Fri 2022-09-09 01:51:19 CST; 5s ago
     Active: inactive (dead)
     Active: active (running) since Fri 2022-09-09 01:51:32 CST; 6s ago
/etc/systemd/system/multi-user.target.wants/corosync-qnetd.service
Removed /etc/systemd/system/multi-user.target.wants/corosync-qnetd.service.
find: ‘/etc/systemd/system/multi-user.target.wants/corosync-qnetd.service’: No such file or directory
Failed to reload corosync-qnetd.service: Job type reload is not applicable for unit corosync-qnetd.service.
     Active: active (running) since Fri 2022-09-09 01:51:32 CST; 15s ago
Fri Sep  9 01:51:48 2022 - INFO  - Finish test!
Fri Sep  9 01:51:49 2022 - INFO  - succeed to execute the case.
Fri Sep  9 01:51:49 2022 - INFO  - start environment cleanup.
oe_test_service_corosync-qnetd.sh: line 55: pcs: command not found
Fri Sep  9 01:51:52 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Traceback (most recent call last):
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:03:26 ago on Fri 09 Sep 2022 01:47:09 AM CST.\nPackage corosync-3.0.4-1.oe2209.riscv64 is already installed.\nError: \n Problem: package pacemaker-2.0.5-1.oe2209.riscv64 requires resource-agents, but none of the providers can be installed\n  - package resource-agents-4.2.0-3.oe2209.riscv64 requires cifs-utils, but none of the providers can be installed\n  - conflicting requests\n  - nothing provides libwbclient.so.0()(64bit) needed by cifs-utils-6.14-3.oe2209.riscv64\n  - nothing provides libwbclient.so.0(WBCLIENT_0.9)(64bit) needed by cifs-utils-6.14-3.oe2209.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
Fri Sep  9 01:51:56 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Failed to stop pacemaker.service: Unit pacemaker.service not loaded.
Failed to stop pcsd.service: Unit pcsd.service not loaded.
Traceback (most recent call last):
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:03:26 ago on Fri 09 Sep 2022 01:47:09 AM CST.\nPackage corosync-3.0.4-1.oe2209.riscv64 is already installed.\nError: \n Problem: package pacemaker-2.0.5-1.oe2209.riscv64 requires resource-agents, but none of the providers can be installed\n  - package resource-agents-4.2.0-3.oe2209.riscv64 requires cifs-utils, but none of the providers can be installed\n  - conflicting requests\n  - nothing provides libwbclient.so.0()(64bit) needed by cifs-utils-6.14-3.oe2209.riscv64\n  - nothing provides libwbclient.so.0(WBCLIENT_0.9)(64bit) needed by cifs-utils-6.14-3.oe2209.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
Failed to start firewalld.service: Unit firewalld.service not found.
Failed to enable unit: Unit file firewalld.service does not exist.
Fri Sep  9 01:52:03 2022 - ERROR - You need to check the environment configuration file to see if this node information exists.
Fri Sep  9 01:52:03 2022 - INFO  - Finish environment cleanup!
